dist/index.html

```diff
@@ -2,19 +2,218 @@
+              <th class="sort cost">Cost</th>
+            </thead>
+
+            <!-- APPEND ALL TRIPS  -->
+            <tbody id="trip-list">
+            </tbody>
+          </table>
+        </div>
+      </section>
+
+    </div> <!-- Content Wrapper -->
   </main>
-  <footer>
 
+  <footer class="align-center">
+    <p>
+      COPYRIGHT &copy; ID 2017
+    </p>
   </footer>
+
+  <!-- ALL TRIP TEMPLATE -->
+
+  <script id="trip-template" type="text/template">
+    <tr id="trip<%-id%>">
+      <td>
+        <%- id %>
+      </td>
+      <td>
+        <%- name %>
+      </td>
+      <td>
+        <%- continent %>
+      </td>
+      <td>
+        <%- category %>
+      </td>
+      <td>
+        <%- weeks %>
+      </td>
+      <td>
+        <%- cost %>
+      </td>
+    </tr>
+  </script>
+
+  <!-- TRIP TEMPLATE -->
+
+  <script id="trip-info-template" type="text/template">
+    <section class="padding align-center">
+      <header>
+        <h1>
+          <%- name %>
+        </h1>
+        <small>
+          <%- continent %>
+        </small>
+      </header>
+
+      <section>
+        <p>
+          <%- about %>
+        </p>
+      </section>
+    </section>
+
+    <div id="reserve-wrapper">
+      <header>
+        <h4 class="header-styling cardo-font align-center">Please enter your information below</h4>
+      </header>
+
+      <form id="reserve-form" class="small-11 small-centered columns cardo-font padding">
+        <!-- <small class="error">FIELDS CANNOT BE BLANK</small> -->
+        <input type="hidden" name="tripId" value="<%-id%>"></input>
+
+        <label>Name:</label>
+        <input type="text" id="name" name="name"></input>
+
+        <label>Age:</label>
+        <input type="number" id="age" name="age"></input>
+
+        <label>Email:</label>
+        <input type="text" id="email" name="email"></input>
+
+        <button class="button to-reserve-black" type="submit">Reserve</button>
+      </form>
+    </div>
+  </script>
+
+  <!-- ERROR TEMPLATE -->
+  <script id="error-template" type= "text/template">
+    <section class="alert callout error align-center">
+      <p>
+        <% if (obj.name) { %> Name: <%- name %><% } %>
+      </p>
+
+      <p>
+        <% if (obj.email) { %> Email: <%- email %><% } %>
+      </p>
+
+      <p>
+        <% if (obj.continent) { %> Continent: <%- continent %><% } %>
+      </p>
+
+      <p>
+        <% if (obj.category) { %> Category: <%- category %><% } %>
+      </p>
+
+      <p>
+        <% if (obj.weeks) { %> Weeks: <%- weeks %><% } %>
+      </p>
+
+      <p>
+        <% if (obj.cost) { %> Cost: <%- cost %><% } %>
+      </p>
+
+      <button class="close-button" aria-label="Dismiss alert" type="button">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </section>
+
+  </script>
+
   <script src="app.bundle.js" type="text/javascript"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/foundation/6.4.3/js/foundation.js"></script>
```

Comment:
By adding foundation here you're getting an error because jQuery isn't defined in global scope.  to get rid of it you'd need to link in jQuery via a script tag, or use the foundation-sites npm package.